js/MVP/views/landing/Landing.js

```diff
@@ -0,0 +1,147 @@
+          stund.
+        </AlertStripeStyled>
+      );
+    }
+
+    return null;
+  };
+
+  const harSammeAvlysningsstatus = (brevType, moteplanleggerStatus) => {
+    return (
+      (brevType === brevTypes.AVLYST && moteplanleggerStatus === AVBRUTT) ||
+      (brevType !== brevTypes.AVLYST && moteplanleggerStatus !== AVBRUTT)
+    );
+  };
+
+  const displayBrev = () => {
+    if (brev.isError || brev.data.length === 0) {
+      return false;
+    }
+
+    if (!moteplanlegger.isError && moteplanlegger.data) {
+      const brevDatoArraySorted = brev.data.map((i) => new Date(i.createdAt)).sort((a, b) => b - a);
+      const sistOpprettetBrev = brevDatoArraySorted[0];
+      const sistOpprettetMoteplanleggerMoteTidspunkt = new Date(moteplanlegger.data.opprettetTidspunkt);
+
+      if (harSammeAvlysningsstatus(sistOpprettetBrev.brevType, moteplanlegger.data.status)) {
+        return sistOpprettetBrev > sistOpprettetMoteplanleggerMoteTidspunkt;
+      }
+    }
+    return true;
+  };
+
+  const displayMotebehov = () => {
+    if (motebehov.isError || !motebehov.data.visMotebehov) return false;
+    if (!moteplanlegger.isError && moteplanlegger.data.status !== AVBRUTT && !erMotePassert(moteplanlegger.data))
+      return false;
+    if (!brev.isError && brev.data[0]) {
+      const brevHead = brev.data[0];
+      if (brevHead.brevType === brevTypes.INNKALT || brevHead.brevType === brevTypes.ENDRING) return false;
+    }
+
+    return true;
+  };
+
+  const BrevPanel = () => {
+    const brevHead = brev.data[0];
+
+    if (brevHead.brevType === brevTypes.REFERAT) {
+      const date = getLongDateFormat(brevHead.tid);
+      return <MotereferatPanel date={date} />;
+    }
+    return <MoteinnkallelsePanel innkallelse={brevHead} />;
+  };
+
+  const PlanleggerPanel = () => {
+    const modus = getSvarsideModus(moteplanlegger.data, BRUKER);
+    const convertedMotedata = konverterTid(moteplanlegger.data);
+    if (modus === BEKREFTET || modus === MOTESTATUS) {
+      return <MoteplanleggerKvitteringPanel mote={convertedMotedata} modus={modus} />;
+    }
+    return <MoteplanleggerPanel modus={modus} />;
+  };
+
+  const DialogmoteFeaturePanel = () => {
+    if (displayBrev()) {
+      return BrevPanel();
+    }
+    if (!moteplanlegger.isError && !erMotePassert(moteplanlegger.data)) {
+      return PlanleggerPanel();
+    }
+    return null;
+  };
+
+  const PreviousMotereferatFeaturePanel = () => {
+    if (brev.isError || brev.data.length < 2) return null;
+
+    const currentBrev = displayBrev() ? brev.data.slice(1) : brev.data;
+    const previousReferater = currentBrev.filter((hendelse) => hendelse.brevType === brevTypes.REFERAT);
+    const previousReferatDates = previousReferater.map(({ tid }) => tid);
+
+    return <PreviousMotereferatPanel previousReferatDates={previousReferatDates} />;
+  };
+
+  return (
+    <DialogmoteContainer title="Dialogmøter">
+      <VeilederLanding />
+
+      <FetchFailedError />
```

Comment:
Trenger ikke gjøre noe med det nå, men det går også an å ha en global feilmeldings-bar som viser dersom react query får noen feil. Ala:
https://github.com/navikt/syfooversikt/blob/master/src/components/error/NotificationBar.tsx

..og i bruk:
https://github.com/navikt/syfooversikt/blob/master/src/data/personoversiktHooks.ts#L42

Baserer seg på React Context.

Reply:
Veldig for det!